src/api/http/httpRequest.ts

```diff
@@ -0,0 +1,17 @@
+import { AxiosInstance } from 'axios';
+
+export class HttpRequest {
+  private service: AxiosInstance;
+
+  constructor(service: AxiosInstance) {
+    this.service = service;
+  }
+
+  get<T extends unknown>(url: string) {
+    return this.service.get<T>(`${url}`).then((res) => res);
```

Comment:
템플릿 리터럴을 제외할 수 있을까요?
```suggestion
    return this.service.get<T>(url).then((res) => res);
```

Reply:
아 수정해야겠네요 감사합니다